docs2/site/docs/migrations/migration5.md

```diff
@@ -150,6 +150,63 @@ and data loader methods are supported such as shown in the below example:
 The above example assumes that the `Person` CLR type was mapped to `PersonGraphType`
 in the schema CLR type mappings.
 
+Arguments are also added as query arguments to the field, and recognize default values
+and other attributes set on the parameter. Certain argument types are recognized and
+treated special; special fields do not add a query argument to the field:
+
+| Argument type                     | Value passed to the method   |
+|-----------------------------------|------------------------------|
+| `IResolveFieldContext`            | The field resolver's context |
+| `CancellationToken`               | The cancellation token from the resolve context |
```

Comment:
Maybe add a note that "injecting" `CancellationToken` may be a common case for methods that don't have dependencies from GraphQL.NET and "injecting" `IResolveFieldContext` brings such dependency and should be treated as "advanced" usage?

Reply:
Like: `The field resolvers's context; useful in advanced scenarios` ?

Reply:
Something like that, yes. Techically we can inject any property from `IResolveFieldContext` but you added only `CancellationToken` . Why? Because its simplicity and common usage.

Reply:
Added as I mentioned above.